mlflow/utils/file_utils.py

```diff
@@ -483,3 +484,21 @@ def _handle_readonly_on_windows(func, path, exc_info):
         raise exc_value
     os.chmod(path, stat.S_IWRITE)
     func(path)
+
+
+def _get_code_dirs(src_code_path, dst_code_path=None):
```

Comment:
Looks like this was left over after the migration to `model_utils`. Can we revert changes to `file_utils`?